portfolio/src/main/java/com/google/sps/data/UserStatus.java

```diff
@@ -17,10 +17,25 @@
 /** A UserStatus object to be converted to JSON and sent to the client. */
 public final class UserStatus {
 
-  private final int isLoggedIn;
+  private final Boolean isLoggedIn;
   private final String loginLogoutURL;
 
-  public UserStatus(int isLoggedIn, String loginLogoutURL) {
+  public static enum AuthState {
```

Comment:
Either move this above your instance variables or below the constructor - either make this the first thing in UserStatus or the last thing.